src/ds5/ds5-options.cpp

```diff
@@ -656,4 +656,88 @@ namespace librealsense
             return _uvc_option->is_enabled();
     }
 
+    auto_exposure_limit_option::auto_exposure_limit_option(hw_monitor& hwm, sensor_base* ep)
+        : _hwm(hwm), _sensor(ep)
+    {
+        _range = [this]()
+        {
+            return option_range{ 0, 166666, 1, 0 };
```

Comment:
Max range is 165000 for both Rolling and Global SKUs

Reply:
Corrected.